src/js/views/Projects/Projects.jsx

```diff
@@ -69,7 +69,7 @@ function Projects() {
     } else if (globalState.projects.sort[column] !== direction) {
       sort[column] = direction
     }
-    if (globalState.projects.sort !== sort) {
+    if (JSON.stringify(globalState.projects.sort) !== JSON.stringify(sort)) {
```

Comment:
This is a little fragile since the order of fields is difficult to predict in JS but I _think_ that they should be consistent.  It might be better to look if the column value has changed instead of trying to do a deep comparison here.  Here's an example of what makes me a little nervous:

```javascript
let oldSort = {title: 'asc', score: 'desc'}
let newSort = {title: 'asc', score: 'desc'}
console.log(JSON.stringify(oldSort) === JSON.stringify(newSort)) // true

newSort = Object.fromEntries([['title', 'asc'], ['score', 'desc']])
console.log(JSON.stringify(oldSort) === JSON.stringify(newSort)) // true

newSort = Object.fromEntries([['score', 'desc'], ['title', 'asc']])
console.log(JSON.stringify(oldSort) === JSON.stringify(newSort)) // false
```

AFAICT, the order in which the properties were added is the order in which they will be serialized.

Reply:
> AFAICT, the order in which the properties were added is the order in which they will be serialized 

Correct, that is the expected behavior.

Reply:
actually ... I take that back ... since sort starts off life as a copy of `globalState.projects.sort` and we are only making one change, we are good here.  Please ignore my ramblings.